### DISCUSSION TITLE
Keywords not supported through SDK, or is it?

### DISCUSSION THREAD

**Discussion by tobyurff** [deepgram/community#268] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

Hi there!

We're comparing STT providers at the moment and an essential bit for our use case is that transcription of known first and last name along with locations we know have a high change or happing during the stream is relatively accurate.

In the Node SDK, it's a bit unclear if that works, as there is `keywords` but it doesn't seem to do anything if specified:

```
deepgramStream = deepgram.transcription.live({
    smart_format: true,
    interim_results: true,
    language: "en-US",
    model: "nova",
    keywords: phrases, 
    keyword_boost: "legacy",
  });
```

Has anyone used this successfully for word hinting/custom vocabulary?

### If possible, please attach your code or paste it into the text box.

deepgramStream = deepgram.transcription.live({
    smart_format: true,
    interim_results: true,
    language: "en-US",
    model: "nova",
    keywords: phrases, 
    keyword_boost: "legacy",
  });

**Comment by jjmaldonis** [comment, marked as answer]
Hey @tobyurff, do you have spaces in your keywords? If so, try removing the spaces. You can also boost words more significantly by adding a `:<number>` after the keyword, such as `Jason:3000`. `3000` is _extremely_ high, and numbers between 1-10 are more typical. You'll want to boost words higher if they have similar pronunciation to other words in your audio.

Also, is there a reason you're using `keyword_boost: legacy`? It sounds like you are most interested in boosting out-of-vocab keywords (names and places), so you will want to use the latest version of keyword boosting.
